sdk/cosmos/azure-cosmos/azure/cosmos/_cosmos_client_connection.py

```diff
@@ -2418,11 +2889,17 @@ def __Patch(self, path, request_params, request_data, req_headers, **kwargs):
             **kwargs
         )
 
-    def __Delete(self, path, request_params, req_headers, **kwargs):
+    def __Delete(
+        self,
+        path: str,
+        request_params: RequestObject,
+        req_headers: Mapping[str, Any],
+        **kwargs: Any
+    ) -> Tuple[None, Dict[str, Any]]:
         """Azure Cosmos 'DELETE' http request.
 
         :param str path: the url to be used for the request.
-        :param ~azure.cosmos.RequestObject request_params: the request parameters.
+        :param ~azure.cosmos._request_object.RequestObject request_params: the request parameters.
```

Comment:
should this change be made in the other root CRUD methods as well?

Reply:
Yup. Done.